formats/json/api/kotlinx-serialization-json.api

```diff
@@ -358,6 +358,10 @@ public final class kotlinx/serialization/json/JvmStreamsKt {
 	public static final fun encodeToStream (Lkotlinx/serialization/json/Json;Lkotlinx/serialization/SerializationStrategy;Ljava/lang/Object;Ljava/io/OutputStream;)V
 }
 
+public final class kotlinx/serialization/json/internal/JsonLexerKt {
+	public static final field BATCH_SIZE I
```

Comment:
This one is published api as well

Reply:
I'm more concerned about synthetic <init>